package.json

```diff
@@ -97,7 +97,7 @@
   "dependencies": {
     "@elastic/apm-rum": "^5.6.1",
     "@elastic/apm-rum-react": "^1.2.5",
-    "@elastic/charts": "26.0.0",
+    "@elastic/charts": "27.0.0",
```

Comment:
Temporarily added, contains some enhancements on small multiples that haven't released yet on kibana